misaligned.cpp

```diff
@@ -1,21 +1,49 @@
 #include <iostream>
 #include <assert.h>
+#include<string>
+#include "misaligned.hpp"
+using namespace std;
 
+colorInfo colorInfoobject2;
+void colorInfo::GetColorCodeManualRef(){
+    int PNumber = 1;
+    for(int i_majorClrCount = 0; i_majorClrCount < numberOfMajor; i_majorClrCount++){
+        for(int j_minorClrCount = 0; j_minorClrCount < numberOfMinor; j_minorClrCount++){
+            PairNumber.push_back(PNumber);
+            MajorColor.push_back(MajorColorNames[i_majorClrCount]);
+            MinorColor.push_back(MinorColorNames[j_minorClrCount]);
+            PNumber++;
+        }
+    }
+}
+void fillcolorInfoobject(int pairNumber,const char* majorColor,const char* minorColor){
+	colorInfoobject2.PairNumber.push_back(pairNumber);
+    colorInfoobject2.MajorColor.push_back(majorColor);
+    colorInfoobject2.MinorColor.push_back(minorColor);
+}
 int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
     int i = 0, j = 0;
     for(i = 0; i < 5; i++) {

```

Comment:
Use of magic number 5 ! what is the reason for choosing 5 in this expression?
Is there test code to safeguard any changes here?
This could be a confusion point during future maintenance.